profile/README.md

```diff
@@ -3,77 +3,59 @@
 <p align="center">
 <img src="https://github.com/instructlab/.github/blob/main/assets/instructlab-banner.png">
 </p>
+InstructLab is a model agnostic open source AI project that faciliates contributions to Large Language Models (LLMs).
```

Comment:
```suggestion
InstructLab is a model-agnostic open-source AI project that facilitates contributions to Large Language Models (LLMs).
```

Reply:
open source is never hyphenated. This is a hill I will die on. :) will fix model-agnostic.

Reply:
- this project is open source. (not hyphenated as a noun)
- open-source software (hyphenated as an adjective)

https://www.grammarly.com/blog/hyphen/

This isn't about open source specifically, but just grammar ... but it's not a hill I will die on.